src/Microsoft.PowerShell.Commands.Utility/commands/utility/WebCmdlet/ConvertToJsonCommand.cs

```diff
@@ -99,23 +99,7 @@ protected virtual void Dispose(bool disposing)
                 _cancellationSource.Dispose();
             }
         }
-
-        /// <summary>
-        /// Prerequisite checks.
-        /// </summary>
-        protected override void BeginProcessing()
-        {
-            if (_depth > maxDepthAllowed)
-            {
-                string errorMessage = StringUtil.Format(WebCmdletStrings.ReachedMaximumDepthAllowed, maxDepthAllowed);
```

Comment:
The resource string `ReachedMaximumDepthAllowed` should be removed. It's only used in `ConvertTo-Json`.
The resource string can be found in `src\Microsoft.PowerShell.Commands.Utility\resources\WebCmdletStrings.resx`